contrib/babelfishpg_tds/src/backend/tds/tds_srv.c

```diff
@@ -187,7 +187,8 @@ pe_tds_init(void)
 	pltsql_plugin_handler_ptr->stmt_end = TDSStatementEndCallback;
 	pltsql_plugin_handler_ptr->stmt_exception = TDSStatementExceptionCallback;
 	pltsql_plugin_handler_ptr->send_column_metadata = SendColumnMetadata;
-	pltsql_plugin_handler_ptr->get_mapped_error_list = &get_mapped_error_code_list;
+	pltsql_plugin_handler_ptr->get_mapped_error_list = &get_mapped_error_list;
+	pltsql_plugin_handler_ptr->get_mapped_tsql_error_list = &get_mapped_tsql_error_code_list;
```

Comment:
should be called get_mapped_tsql_error_code_list to avoid confusion

Reply:
Done